input/drivers/ps2_input.c

```diff
@@ -1,6 +1,5 @@
 /*  RetroArch - A frontend for libretro.
- *  Copyright (C) 2010-2014 - Hans-Kristian Arntzen
- *  Copyright (C) 2011-2017 - Daniel De Matteis
+ *  Copyright (C) 2011-2018 - Francisco Javier Trujillo Mata - fjtrujy
```

Comment:
Why are you removing the Copyright information?

Reply:
I forgot to do this in the previous PR.
This file was fully created by me.
The problem was that in order to start I duplicated the content of the PSP1 files.
But in the end, I didn't use anything from those files.